test/populateTestData.js

```diff
@@ -2,6 +2,7 @@ const { db, client } = require('../src/database/connection');
 const testStatuses = require('./fixtures/testBulkStatus.json');
 const testOperationOutcome = require('./fixtures/fhir-resources/testOperationOutcome.json');
 const queue = require('../src/queue/importQueue');
```

Comment:
```suggestion
const { importQueue } = require('../src/queue/importQueue');
```

Reply:
Now that we have more than one queue we should probably be more specific with the import queue var name.

Reply:
Addressed in f429040.